lib/tests/release.nix

```
@@ -24,7 +24,10 @@ in
       #
       #   https://github.com/NixOS/nixpkgs/issues/272591
       #
-      [(import ../../pkgs/test/release {})]
+      [(import ../../pkgs/test/release {
+        inherit pkgs lib;
+        pkgs-path = pkgs.path;
+      })]
```

I don't think `pkgs-path` has to be set anymore (and `pkgs.path` is a bit of an odd-ball anyways). Furthermore, I'm just seeing that @lf- opened https://github.com/NixOS/nixpkgs/pull/318654 which also changes this part, notably also setting Nix, which I think is good, so:

```suggestion
      [(import ../../pkgs/test/release {
        inherit pkgs lib nix;
      })]
```

---

Alright, removed `pkgs-path`.